### ISSUE TITLE
COCO training codes

### ISSUE THREAD

**Issue by pigcv** [halbielee/EPS#6]
Hi~ Could you provide COCO training codes, to help us reproduce the results on COCO?

Thanks!

**Comment by halbielee** [comment]
Hello, @pigcv  again!

I will provide the training code & dataset for COCO soon!

**Comment by pigcv** [comment]
> Hello, @pigcv again!
> 
> I will provide the training code & dataset for COCO soon!

Thanks for you reply. Another question is that why deeplab v1 performs better than deeplab v2? Which Deeplab V1 repo are you using? Thanks again!

**Comment by halbielee** [comment]
We basically use the network in [here](https://github.com/kazuto1011/deeplab-pytorch).

But it only provides DeepLab-V2. Since the backbone networks are the same between DeepLabV1 & V2, we manually make DeepLabV1 from DeepLabV2. We use the same pre-trained model!

**Comment by halbielee** [comment]
Hello, @pigcv 

I uploaded the training data for MS-COCO 2014 (segmentation mask for train/validation set) and the training code.
You can check them with new script files.

Also, I updated the checkpoints, training logs, performances for each method and each dataset.

Please see [README.md](https://github.com/halbielee/EPS/blob/main/README.md)!

**Comment by pigcv** [comment]
> Hello, @pigcv
> 
> I uploaded the training data for MS-COCO 2014 (segmentation mask for train/validation set) and the training code.
> You can check them with new script files.
> 
> Also, I updated the checkpoints, training logs, performances for each method and each dataset.
> 
> Please see [README.md](https://github.com/halbielee/EPS/blob/main/README.md)!

Thanks very much!
